### PR TITLE
Update jans.txt

### PR DESCRIPTION
**Pull request by archanathammali1252**
updated

### CODE REVIEW

File: jans.txt [hunk 1]
```diff
@@ -1 +1 @@
-hello
\ No newline at end of file
+hello hans
```
Comment (ramure) [inline jans.txt:1]: please update

**Comment by archanathammali1252** [comment]
hey

File: jans.txt (hunk 1 above)
Comment (ramure) [reply jans.txt:1]: pls

Comment (archanathammali1252) [reply jans.txt:1]: ok

**Pull request merged**